src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -41,19 +41,39 @@ protected ActionResult execute() throws EntityDoesNotExistException {
 
         AdminHomePageData data = new AdminHomePageData(account);
 
-        data.instructorShortName = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_SHORT_NAME);
-        Assumption.assertNotNull(data.instructorShortName);
-        data.instructorName = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_NAME);
-        Assumption.assertNotNull(data.instructorName);
-        data.instructorEmail = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_EMAIL);
-        Assumption.assertNotNull(data.instructorEmail);
-        data.instructorInstitution = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_INSTITUTION);
-        Assumption.assertNotNull(data.instructorInstitution);
+        data.instructorDetailsSingleLine = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_DETAILS_SINGLE_LINE);
+        data.instructorShortName = "";
+        data.instructorName = "";
+        data.instructorEmail = "";
+        data.instructorInstitution = "";
+        
+        if (data.instructorDetailsSingleLine != null) {
```

Comment:
Actually, maybe add a comment or extract a variable to explain this, explaining that the single line form takes precedence over the other form